### ISSUE TITLE
error: 'NSS_IPV4_RULE_CREATE_RAWIP_VALID' undeclared (first use in this function); 

### ISSUE THREAD

**Issue by kuncy7** [qosmio/nss-packages#32]
This is how today's compilation ends:
```
 -DECM_OPENWRT_SUPPORT=1 -DECM_NOTIFIER_DEBUG_LEVEL=1 -DECM_AE_CLASSIFIER_DEBUG_LEVEL=1 -DECM_STATS_DEBUG_LEVEL=1 -DECM_CLASSIFIER_WIFI_DEBUG_LEVEL=1 -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/ -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/ecm_db -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/include -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/sfe -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/cmn -I/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/exports -Wall -Werror -I/home/metium/openwrt-ipq/staging_dir/target-aarch64_cortex-a53_musl/usr/include/qca-nss-drv -I/home/metium/openwrt-ipq/staging_dir/target-aarch64_cortex-a53_musl/usr/include/qca-mcs -I/home/metium/openwrt-ipq/staging_dir/target-aarch64_cortex-a53_musl/usr/include/nat46 -Wno-error=unused-function  -DMODULE  -DKBUILD_BASENAME='"ecm_nss_non_ported_ipv4"' -DKBUILD_MODNAME='"ecm"' -D__KBUILD_MODNAME=kmod_ecm -c -o /home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.o /home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.c  
../qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.c: In function 'ecm_nss_non_ported_ipv4_connection_accelerate':
../qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.c:859:47: error: 'NSS_IPV4_RULE_CREATE_RAWIP_VALID' undeclared (first use in this function); did you mean 'NSS_IPV4_RULE_CREATE_RPS_VALID'?
  859 |                         nircm->valid_flags |= NSS_IPV4_RULE_CREATE_RAWIP_VALID;
      |                                               ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                               NSS_IPV4_RULE_CREATE_RPS_VALID
../qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.c:859:47: note: each undeclared identifier is reported only once for each function it appears in
make[6]: *** [scripts/Makefile.build:243: /home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/frontends/nss/ecm_nss_non_ported_ipv4.o] Error 1
make[5]: *** [/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/linux-6.6.50/Makefile:1921: /home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1] Error 2
make[4]: *** [Makefile:234: __sub-make] Error 2
make[4]: Leaving directory '/home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/linux-6.6.50'
make[3]: *** [Makefile:191: /home/metium/openwrt-ipq/build_dir/target-aarch64_cortex-a53_musl/linux-qualcommax_ipq807x/qca-nss-ecm-12.5.2024.06.20~92c7fa1/.built] Error 2
make[3]: Leaving directory '/home/metium/openwrt-ipq/feeds/nss_packages/qca-nss-ecm'
time: package/feeds/nss_packages/qca-nss-ecm/compile#1.91#0.23#2.12
    ERROR: package/feeds/nss_packages/qca-nss-ecm failed to build.
make[2]: *** [package/Makefile:177: package/feeds/nss_packages/qca-nss-ecm/compile] Error 1
make[2]: Leaving directory '/home/metium/openwrt-ipq'
make[1]: *** [package/Makefile:171: /home/metium/openwrt-ipq/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/metium/openwrt-ipq'
make: *** [/home/metium/openwrt-ipq/include/toplevel.mk:248: world] Error 2
```

**Comment by derzahla** [comment]
yeh me 2!

**Comment by qosmio** [comment]
Please check latest fix aa57d9c4820559021f2b0d7db9496e28bf503058

**Comment by kuncy7** [comment]
Thank you, now compilation completes successfully without error.
